proto/spacemesh/v1/smesher_types.proto

```diff
@@ -127,4 +127,21 @@ message PostSetupStatus {
     uint64 num_labels_written = 2; // Number of labels (hashes) written to the data files
     PostSetupOpts opts = 3; // setup options previously set by the user
     string error_message = 4; // The error message, if the state is STATE_ERROR
+    string processed = 5; // The processed size, like `176 GB`
+    string post_total_size = 6; // Total size, that should be processed, like `256 GB`
+    uint32 processed_percentage = 7; // The processed percentage, like `50`
+}
+
+message SmeshingStatusResponse {
```

Comment:
Please break out the rest of this message into its own message type called `SmeshingStatus`. See how `PostSetupStatus` is implemented here. Please follow the existing design.